esp/bindings/http/client/httpclient.hpp

```diff
@@ -41,6 +41,7 @@ interface IHttpClient : extends ITransportClient
 
     virtual void setConnectTimeOutMs(unsigned timeout) = 0;
     virtual void setTimeOut(unsigned int timeout) = 0;
+    virtual void setTxSummary(CTxSummary* txSummary) = 0;
```

Comment:
Is the visibility of CTxSummary guaranteed here? I might forward declare the class just to be safe.

Reply:
There is already a forward declaration in esp.hpp included in soapesp.hpp which is included in httpclient.hpp.